github/admin_users.go

```diff
@@ -59,3 +60,74 @@ func (s *AdminService) DeleteUser(ctx context.Context, username string) (*Respon
 
 	return resp, nil
 }
+
+// ImpersonateUserOptions represents the scoping for the OAuth token.
+type ImpersonateUserOptions struct {
+	Scopes *[]string `json:"scopes,omitempty"`
```

Comment:
Throughout this PR, please change all instances of `*[]string` to `[]string` as the slice is already a reference type.